Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabInstanceUtils.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/string/string.h>
+#include <AzCore/std/utils.h>
+
+namespace AzToolsFramework
+{
+    namespace Prefab
+    {
+        class Instance;
+        namespace PrefabInstanceUtils
+        {
+            /**
```

Comment:
nit : use Doxygen style markup for comments and add comments about @param and @return values